mappings/net/minecraft/block/Block.mapping

```diff
@@ -1,19 +1,30 @@
 CLASS none/akw net/minecraft/block/Block
 	FIELD A stateFactory Lnone/ask;
 	FIELD a id Lnone/kp;
+	FIELD b creativeTab Lnone/adq;
 	FIELD c defaultState Lnone/asj;
+	FIELD d unlocalizedName Ljava/lang/String;
```

Comment:
*shrugs* I went with `unlocalizedName` because that's what it is, the block's name before it's been translated for the current locale.